jobs/integr8ly/ocp4/ocp4-install.yaml

```diff
@@ -168,13 +192,19 @@
                                 sed -i s@INTEGREATLY-OPERATOR_NAMESPACE@${namespaceName}@g ./integreatly-qe/resources/integreatly-operator/operatorgroup-cr.yaml
                                 sed -i s@INTEGREATLY-OPERATOR_NAMESPACE@${namespaceName}@g ./integreatly-qe/resources/integreatly-operator/subscription-cr.yaml
 
-                                ./oc create -f ./integreatly-qe/resources/integreatly-operator/catalogsourceconfig-cr.yaml -n openshift-marketplace
-                                ./oc create -f ./integreatly-qe/resources/integreatly-operator/operatorgroup-cr.yaml -n ${namespaceName}
-                                ./oc create -f ./integreatly-qe/resources/integreatly-operator/subscription-cr.yaml -n ${namespaceName}
+                                oc create -f ./integreatly-qe/resources/integreatly-operator/catalogsourceconfig-cr.yaml -n openshift-marketplace
+                                oc create -f ./integreatly-qe/resources/integreatly-operator/operatorgroup-cr.yaml -n ${namespaceName}
+                                oc create -f ./integreatly-qe/resources/integreatly-operator/subscription-cr.yaml -n ${namespaceName}
                             """
                         )
                     } // stage
 
+                    stage ('Create GitHub secret') {
+                        withCredentials([string(credentialsId: "${githubSecretId}", variable: 'SECRET')]) {
+                            sh "oc create secret generic github-oauth-secret --from-literal=clientId=${githubSecretId} --from-literal=secret=${SECRET} -n ${namespaceName}"
```

Comment:
Just a question again, is it ok for this GH client ID and GH client secret to be the same for all the clusters? No idea how it works for intly on OCP4 but for intly on OCP3 the callback URL is unique for each cluster so two different clusters could not use the same GH OAuth app.

Reply:
I think the question really is whether to parametrize GH client ID and GH client Secret or there is some difference for intly on OCP4 and it is no longer needed.